src/label-checker.ts

```
@@ -13,28 +14,8 @@ export class LabelChecker {
-			throw new Error(
-				'Invalid repository name. Repository names must be between 1-100 characters, start with a letter/number, and can contain hyphens.',
-			)
-		}
-		if (!octokit) throw new Error('Octokit instance is required')
-
-		this.owner = trimmedOwner
-		this.repo = trimmedRepo
-		this.octokit = octokit
+	constructor(octokit: Octokit, repoPath: string) {
+		super(octokit, repoPath)
 	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)